tools/bootstrap_upip.sh

```diff
@@ -0,0 +1,13 @@
+# This script performs bootstrap installation of upip package manager from PyPI
+# All the other packages can be installed using it.
+
+# Remove any stale old version
+rm -rf micropython-upip-*
```

Comment:
I think it's too dangerous to have such a command in a bash script that people will blindly run anywhere on their system.  Can we wrap it in a check that such a directory exists?